packages/editor/src/browser/editor-navigation-contribution.ts

```diff
@@ -188,6 +193,14 @@ export class EditorNavigationContribution implements Disposable, FrontendApplica
         }
     }
 
+    /**
+     * Toggle the display of minimap in the editor.
+     */
```

Comment:
Incorrect documentation? We can remove or you should reference "sticky scroll" and not "minimap".

Reply:
Yes, I will correct. thanks a lot!